include/vkgcDefs.h

```diff
@@ -200,6 +200,14 @@ enum ShaderStageBit : unsigned {
   ShaderStageComputeBit = (1 << ShaderStageCompute),         ///< Compute shader bit
 };
 
+/// Enumerates LLPC types of unlinked shader elf.
+enum UnlinkedShaderStage {
```

Comment:
I think this is treated as `unsigned` elsewhere. If that's the case, it'd be better to be explicit here and have `enum UnlinkedShaderStage : unsigned`.